src/services/liveTranscriptionService.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import type {
+	liveTranscriptionDisableResponse,
+	liveTranscriptionEnableResponse,
+	liveTranscriptionGetAvailableLanguagesResponse,
+	liveTranscriptionSetLanguageResponse,
+} from '../types/index.ts'
+
+import axios from '@nextcloud/axios'
+import { generateOcsUrl } from '@nextcloud/router'
+
+/**
+ * Enable live transcription
+ *
+ * @param {string} token conversation token
+ */
+const enableLiveTranscription = async function(token: string): Promise<liveTranscriptionEnableResponse> {
```

Comment:
Should already be a promise
```suggestion
const enableLiveTranscription = async function(token: string): liveTranscriptionEnableResponse {
```